starwars/src/components/SWCard.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import styled from "styled-components";
+
+
+const Card = styled.div`
```

Comment:
Great work here! In the future, remember that you can individually export and import styles from other components if you want to reuse them!